projects/templates/home.html

```diff
@@ -51,9 +53,13 @@
           <form class="d-flex" role="search">
             <input class="form-control me-2" type="search" placeholder="Search the blog" aria-label="Search">
               <i class="bi bi-search">
```

Comment:
need to remove this icon tag **<i class="bi bi-search">**
 those classes are from a different icon package (Blade Icons  i think it is) that is not used in this project
 We are using font awesome